config/locales/activerecord.yml

```diff
@@ -74,7 +74,7 @@ en:
               invalid: Enter an email address in the correct format, like name@example.com
             comment:
               blank: Enter your feedback
-              too_long: Feedback must not be more than %{count} characters
+              too_long: Feedback must not be more than 1,200 characters
```

Comment:
Shame to lose the dynamism, I wonder if `number_with_delimiter` could have helped.

Reply:
Thanks, I can look at trying this later.